setup.cfg

```diff
@@ -0,0 +1,64 @@
+[metadata]
+name = lsst_versions
+description = Calculate package versions following LSST code conventions.
+author = Rubin Observatory Data Management
+url = https://github.com/lsst/lsst_versions
+classifiers =
+    License :: OSI Approved ::  GNU General Public License v3 or later (GPLv3+)
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+version = attr: lsst_versions.__version__
+readme = file: README.md
+
+[options]
+# zip_safe can't be used for mypy compatibility
+zip_safe = False
+package_dir=
+    =python
+packages=find:
+setup_requires =
+  setuptools >=46.0
+install_requires =
+  tomli>1.0.0
+  GitPython >= 3.1
+tests_require =
+  pytest >= 3.2
+  flake8 >= 3.7.5
+  pytest-flake8 >= 1.0.4
+  pytest-openfiles >= 0.5.0
+
+[options.packages.find]
+where=python
+
+[options.package_data]
+lsst_versions = py.typed
```

Comment:
I don't understand. Without this line we couldn't get the wheel to include the `py.typed` file and so mypy usage is disabled. Granted no-one is likely to be using type annotations from this package.

Reply:
I thought that this `package_data` line meant that the packager added the py.typed file to the package, so you didn't need an explicit one in the git repo.

Reply:
It doesn't know what the file contains so it must exist first (I'm pretty sure there isn't special knowledge in setuptools that py.typed is a zero length file)

Reply:
I thought it is special: https://peps.python.org/pep-0561/#packaging-type-information

Reply:
It doesn't quite say it will be created though. I can try it. We have to have it explicitly in utils and resources because scons does not know about it.

Reply:
Oh, but maybe that is just saying to package the file which you had already manually included.  https://docs.python.org/3/distutils/setupscript.html#installing-package-data